src/pseudopeople/utilities.py

```diff
@@ -166,6 +166,19 @@ def get_state_abbreviation(state: str) -> str:
         raise ValueError(f"Unexpected state input: '{state}'") from None
 
 
+# Type aliases for working across multiple engines
+# For now, we don't expose these type aliases to the user --
+# see interface.py for more on this decision.
+ENGINE = Literal["pandas", "modin"]
```

Comment:
I think I'd prefer something like this to avoid needing to use the naked strings "pandas" and "modin" around the codebase. It'd also allow for things like the `numpy_like` and `pandas_like` from your other PR to be defined here.
```
@dataclass
class Engine:
    name: str

PANDAS = Engine("pandas")
MODIN = Engine("modin")
```